### ISSUE TITLE
Reproduce training 

### ISSUE THREAD

**Issue by flpgrz** [IdoSpringer/ERGO-II#8]
Hello,

Could you please document in the repository how to run the training in order to reproduce the paper results?

You only documented how to run 
```commandline
python Predict.py dataset file
```

It would nice to see how to train the models in order to obtain the paper results.

Thanks.

**Comment by flpgrz** [comment]
P.S.

I was doing some experiments, trying to reproduce the paper results and I saw this line:
https://github.com/IdoSpringer/ERGO-II/blob/7a25b90db54252f4691b70af34f0d4f5351941c9/Trainer.py#L235

It seems you do validation on the test set. 
Do you have an additional left-out test set, on which you computed the paper results? Or are the results in the paper computed on the validation set?

Thanks.
